### PR TITLE
Add marpobuda chicago recommendations 

### LINKED ISSUE #9: Add marpobuda Chicago file
- [x] Create a branch
- [x] Add file/commits
- [ ] Open a PR 
- [ ] Have a discussion about the PR 
- [ ] Make edits
- [ ] Get :cat2: :cat2: approval 
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by marpobuda**
close #9 

Hey, @KevinButlerEsri and @j-dacosta , did I miss any good places to eat or to avoid? :city_sunset: 

cc: @beardofedu

### CODE REVIEW

File: marpobuda-Chicago.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Chicago, IL
+
+##Places to Eat
+-Portillos
```
Comment (LLLRRRBBB) [inline marpobuda-Chicago.md]: What kind of food is this?

**Comment by j-dacosta** [comment]
mmm ... :hotdog:

:disappointed: no hotdog emoji!

**Comment by LLLRRRBBB** [comment]
How dare they not have a hot dog emoji!!

**Comment by marpobuda** [comment]
@LLLRRRBBB I added food descriptions, how does this look? :question: :question: :question:

File: marpobuda-Chicago.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+-Spoon (Thai Food)
+-Rolls and Bowls (Sushi)
+
+##Places to Go
+-Sears Tower
+-Ohio Street Beach 
+-Lincoln Square
+
+##Places to :beer:
+
```
Comment (LLLRRRBBB) [inline marpobuda-Chicago.md:14]: You missed the most important part of the list!

**Pull request merged**